pyproject.toml

```diff
@@ -4,30 +4,34 @@
 [tool.bandit]
 exclude_dirs = ["/venv/"]
 [tool.bandit.assert_used]
-skips = ["*/*test.py", "*/test_*.py"]
+skips = ["*/*test.py", "*/test_*.py", "*tests/*.py"]
 
 # Testing tools configuration
 [tool.coverage.run]
 branch = true
 omit = [
     # Contains interface for calling LXD. Tested in integration tests and end to end tests.
     "src/lxd.py",
-    # Contains interface for calling repo policy compliance service. Tested in integration test and end to end tests.
+    # Contains interface for calling repo policy compliance service. Tested in integration test 
+    # and end to end tests.
     "src/repo_policy_compliance_client.py",
 ]
 
 [tool.coverage.report]
-fail_under = 38
+fail_under = 83
```

Comment:
Not directly related to this PR, but we may need a follow-up PR to meet the team standard (requires at least 85): https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-coverage

And thanks for updating this!